src/python/gudhi/reader_utils.cc

```diff
@@ -0,0 +1,108 @@
+/*    This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+ *    See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+ *    Author(s):       Vincent Rouvreau
+ *
+ *    Copyright (C) 2020 Inria
+ *
+ *    Modification(s):
+ *      - YYYY/MM Author: Description of the modification
+ */
+
+#include <gudhi/Points_off_io.h>
+#include <gudhi/reader_utils.h>
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+#include <pybind11/numpy.h>
+#include <pybind11/stl_bind.h>
+
+#include <boost/function_output_iterator.hpp>  // for boost::make_function_output_iterator
```

Comment:
I think it prints a deprecated warning with newer versions of boost.

Reply:
Oh yes you are right, now I remember on your last PR. Done on b7f4cb43